manifests/dynamic-networks-controller.yaml

```diff
@@ -123,3 +124,6 @@ spec:
             items:
               - key: dynamic-networks-config.json
                 path: dynamic-networks-config.json
+        -  name: multus-server-socket
+           hostPath:
+             path: /run/multus/multus.sock
```

Comment:
You could also add `type: Socket` under `hostPath` https://kubernetes.io/docs/concepts/storage/volumes/

That way the Pod will refuse to start if the socket is not found on the host.

Reply:
Done - thanks for the hint.